### ISSUE TITLE
settings page does not reflect selected Theme

### ISSUE THREAD

**Issue by hungryalgo** [huggingface/chat-ui#746]
Settings page is always light/white regardless of the Theme selected (Dark or Light). 

Is this intentional or we just did not have time to respect the selected Theme?

If we need to fix this, how much work load do you expect? Just small change on the main settings page (settings/+layout.svelte) or do we need to change every UI piece in settings?     I might want to fix this if this is not huge.

thanks

**Comment by nsarrazin** [comment]
Yes i think for now that's by design, but I would like to fix it eventually. I think we need to start making components because it's a lot of tailwind classes everywhere atm.
